src/core/hook/wee-hook-url.c

```diff
@@ -220,11 +231,25 @@ hook_url_transfer (struct t_hook *hook)
                          &hook_url_transfer_thread, hook);
     if (rc != 0)
     {
-        gui_chat_printf (NULL,
-                         _("%sError running thread in hook_url: %s (URL: \"%s\")"),
-                         gui_chat_prefix[GUI_CHAT_PREFIX_ERROR],
-                         strerror (rc),
-                         HOOK_URL(hook, url));
+        snprintf (str_error, sizeof (str_error),
+                  "error calling pthread_create (%d)", rc);
```

Comment:
I don't know how to get a human readable string of the pthread_create error code.